server/index.js

```diff
@@ -36,6 +36,24 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:productId', (req, res, next) =>{
+  const { productId } = req.params
+  const sql = `
+          SELECT *
+            FROM "products"
+          WHERE "productId" = ${productId};
```

Comment:
This is a SQL injection vulnerability. Be sure to use parameterized queries when including client-supplied data.